packages/charts/react-charting/src/utilities/utilities.ts

```diff
@@ -1384,3 +1384,22 @@ export const formatDate = (date: Date, useUTC?: boolean) => {
   const timeFormat = useUTC ? d3UtcFormat : d3TimeFormat;
   return timeFormat('%-e %b %Y, %H:%M')(date) + (useUTC ? ' GMT' : '');
 };
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export const getSafeProps = (props: Record<string, any> | undefined) => {
```

Comment:
Can you see if this code is better and works as expected:

function getSecureProps(props: Record<string, unknown> = {}): Record<string, unknown> {
  const { dangerouslySetInnerHTML, ...result } = props;
  return result;
}

Reply:
This looks better. Replaced the implementation.